Packs/Claroty/Integrations/Claroty/Claroty.py

```diff
@@ -0,0 +1,627 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+""" IMPORTS """
+from distutils.util import strtobool
+from typing import List, Tuple, Dict, Any, Optional, Union
+import json
+import requests
+import dateparser
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+DEFAULT_HEADERS = {'content-type': 'application/json'}
+CTD_TO_DEMISTO_SEVERITY = {
+    "Low": 1,
+    "Medium": 2,
+    "High": 3,
+    "Critical": 4
+}
+ALERT_CTD_FIELD_TO_DEMISTO_FIELD = {
+    'resource_id': "ResourceID",
+    'type': "AlertTypeID",
+    'type__': "AlertType",
+    'severity__': "Severity",
+    'network_id': "NetworkID",
+    'resolved': "Resolved",
```

Comment:
does it make sense to add `archived` attribute ?

Reply:
no, the value returned is called resolved. We differentiate by the int return between the two

Reply:
@asaf96 how do you know if something is resolved or archived? The UI shows it. Can you make it show it as "Archived" when it's archived?

Reply:
So I checked and we don't return the type of resolution, only whether it was resolved (resolved or archived) or not